core/src/main/java/org/stellar/anchor/sep24/MoreInfoUrlConstructor.java

```diff
@@ -2,8 +2,9 @@
 
 import java.net.MalformedURLException;
 import java.net.URISyntaxException;
+import org.stellar.anchor.SepTransaction;
 
 public abstract class MoreInfoUrlConstructor {
-  public abstract String construct(Sep24Transaction txn)
+  public abstract String construct(SepTransaction txn)
```

Comment:
Should we move this out of the `sep24` package now that it's being used by SEP-6 and SEP-24?